build/docker/dev/Dockerfile

```diff
@@ -4,7 +4,7 @@ WORKDIR /app
 
 
 RUN apk update \
-    && apk add --no-cache bash make protobuf protobuf-dev git gzip curl
+    && apk add --no-cache bash make protobuf protobuf-dev git gzip curl build-base
```

Comment:
why is build-base needed?

Reply:
I was unable run  (existing / added)  unit tests also without it, not completely sure why its needed.